x/mint/abci_test.go

```diff
@@ -0,0 +1,61 @@
+package mint_test
+
+import (
+	"testing"
```

Comment:
please have imports formatted correctly. If you use vscode, you should have a local config:

```json
{
  "gopls": {
    "local": "github.com/cosmos/cosmos-sdk",
  },
}
```

all imports should be grouped and there should be no newlines between imports within a group.